### PR TITLE
added 'private' label, disabled link, for private repo

### LINKED ISSUE #4250: Private Repository link in repository-overview/frontend-project
### Current Behavior
a private repo is added in the frontend projects, when user clicks on it , he/she couldn't access 

### Desired Situation

- [x] add some label to that repo to indicate that it's a private repo and needs permission

### PR DESCRIPTION
**Pull request by sayyedarib**
**Description**

This PR fixes #4250
private repo link was present in the repo-overview section which was unreachable by the normal user

Before

after fixing
fixed

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by sayyedarib** [comment]
resolving issue #4250

**Comment by l5io** [comment]
🚀 Preview for commit 2719f7b5be749c924899443b3f61af9a199bc668 at: https://646737288f525e5e61d41016--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit fbe4d930c5dec01886d4463001397998ffe459a4 at: https://64673f01e3954f627a0b257e--layer5.netlify.app

**Comment by sayyedarib** [comment]
error
I think error is coming from the path that I haven't edit 
Let me know how to fix this

**Comment by ayushthe1** [comment]
@sayyedarib  Let's discuss this on websites call. Add this as an agenda item in meeting minutes if you would :).
I'll add this item for you if it's ok.

**Comment by sayyedarib** [comment]
Yes sure you can add

**Comment by Savio629** [comment]
You can ignore the lighthouse issue...
The pr looks good to me!

**Review by Savio629 — CHANGES_REQUESTED** [review]
@sayyedarib Just small changes...
Instead of using black color and white background use something dynamic like `${(props) => props.theme.secondaryLightColor}`

**Comment by l5io** [comment]
🚀 Preview for commit c088c7b80b5f8706b6a07618d3f7a34d9d35e71f at: https://6486b6e801d60118f68be802--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit e4ac70935bf579f0acd024df163547117de1ed52 at: https://64897313304b696d9607e78f--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit e4ac70935bf579f0acd024df163547117de1ed52 at: https://648973e5304b696e1b07e912--layer5.netlify.app

File: src/sections/Community/Handbook/Handbook.style.js [hunk 1]
```diff
@@ -131,6 +131,24 @@ export const HandbookWrapper = styled.div`
       text-align: left;
       padding: 0.5rem;
       transition: 0.8s cubic-bezier(0.2, 0.8, 0.2, 1);
+
```
Comment (sayyedarib) [inline src/sections/Community/Handbook/Handbook.style.js:134]: styling of label

File: src/sections/Community/Handbook/repository.js [hunk 2]
```diff
@@ -159,8 +167,18 @@ const Repository = () => {
                                 href={repository}
                                 target="_blank"
                                 rel="noreferrer"
+                                className={
```
Comment (sayyedarib) [inline src/sections/Community/Handbook/repository.js:170]: if repo name is meshery cloud then 
add private label

**Comment by l5io** [comment]
🚀 Preview for commit ac2b71d2a75774e2b746515bb5e77c98fcb5fb9b at: https://6489aa28f2277a19173015de--layer5.netlify.app

File: src/sections/Community/Handbook/Handbook.style.js [hunk 3]
```diff
@@ -131,6 +131,24 @@ export const HandbookWrapper = styled.div`
...
+      pointer-events: none;
+      cursor: default;
+      opacity: 0.7;
+      align-items:center;
+    }
+    .label {
+      position:relative;
+      border-radius: 0.5rem;
+      font-size: 0.8rem;
+      background: ${(props) => (props.theme.DarkTheme ? "white" : "black")};
```
Comment (sayyedarib) [inline src/sections/Community/Handbook/Handbook.style.js:147]: change labe background and text with theme

**Comment by sayyedarib** [comment]
I have commented on the lines where there's actual changes @Savio629 
I think I have messed up 😄

**Comment by l5io** [comment]
🚀 Preview for commit 3f1b051b6defab58d7505f70b91c4b0bd6a6f4a9 at: https://6489adfc8fc6141ef77ef6a0--layer5.netlify.app

File: src/sections/Community/Handbook/Handbook.style.js (hunk 3 above)
Comment (Savio629) [reply src/sections/Community/Handbook/Handbook.style.js:147]: ${props => props.theme.whiteToBlack};
This should work!

**Comment by l5io** [comment]
🚀 Preview for commit 68f7019f1e9843db81a17a7699e284c281faae35 at: https://648aa596f80521460f9d4028--layer5.netlify.app

**Review by Savio629 — APPROVED** [review]

**Review by DarrenDsouza7273 — APPROVED** [review]
Good work @sayyedarib

**Review by thisiskaransgit — CHANGES_REQUESTED** [review]
Screenshot from 2023-06-17 17-22-47
- Align the text in the center
- Please do not change the code style. 

 Also, while you're on the page, let's also change the reference of Meshmap to https://layer5.io/cloud-native-management/meshmap, currently, it points to a preview of a  commit 
// @Savio629 @DarrenDsouza7273

**Comment by sayyedarib** [comment]
> Screenshot from 2023-06-17 17-22-47
> 
> * Align the text in the center
> * Please do not change the code style.
> 
> Also, while you're on the page, let's also change the reference of Meshmap to https://layer5.io/cloud-native-management/meshmap, currently, it points to a preview of a commit // @Savio629 @DarrenDsouza7273

@thisiskaransgit 
it looks fine at my end
see this preview: https://648aa596f80521460f9d4028--layer5.netlify.app/


are you talking about this reference?

**Comment by Savio629** [comment]
@sayyedarib  Let's discuss this on websites call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.69qskjv558bk) if you would :)

**Comment by sayyedarib** [comment]
> @sayyedarib,
> 
> * I must have seen the older preview, the change looks good
> * Disable your current lint extension and format it again.
> * Yes, update the link for Meshmap

-I have updated the link
-But i don't know how to revert the formatting, I have disabled the prettier and using the default formatting
still it didn't reverted back to normal

**Comment by sayyedarib** [comment]
> @sayyedarib Let's discuss this on websites call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.69qskjv558bk) if you would :)

sorry for late reply, 
missed the meet :(

**Comment by l5io** [comment]
🚀 Preview for commit f178111f491423b4751a689fa32a4507f1e7040d at: https://6492929e3e57481c82de8f55--layer5.netlify.app

**Comment by Savio629** [comment]
This pr fixes #4250 
Do remember to link your issue with the pr.
Regarding the formatting changes I would suggest if there are few lines that were formatted due to the extension, you can simply copy the previous code and paste it.
Also we have weekly website meetings on every monday at 5.30 pm IST
// @thisiskaransgit

**Comment by l5io** [comment]
🚀 Preview for commit 0177afb535508c1b4fea19edb7db44b2de6b4a1e at: https://6498616ae163ac1e7ee8f636--layer5.netlify.app

**Comment by vishalvivekm** [comment]
@sayyedarib Let's discuss this in websites call. Add this as agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.69qskjv558bk) if you would :)

**Comment by thisiskaransgit** [comment]
Hey, @sayyedarib, wanted to check in on the progress. Did you incorporate the feedback that was given in the last meeting, do let us know if you have any questions, and let's wrap up this task.

**Comment by sayyedarib** [comment]
I had not attended the meet but whatever changes that were requested in conversations of github is done

**Comment by thisiskaransgit** [comment]
@sayyedarib, the recorded meeting link has been shared by @Savio629, 2 weeks ago

**Comment by sayyedarib** [comment]
> @sayyedarib, the recorded meeting link have been shared by @Savio629, 2 weeks ago

i think my PR was not discussed although it was added as i was not available also the time was over

**Comment by Savio629** [comment]
@sayyedarib Please do watch it properly
https://www.youtube.com/live/D-cKaUrnQo4?feature=share
31:09 There was a discussion regarding your issue at mentioned timeframe.

**Comment by sayyedarib** [comment]
> @sayyedarib Please do watch it properly https://www.youtube.com/live/D-cKaUrnQo4?feature=share 31:09 There was a discussion regarding your issue at mentioned timeframe.

got it thank you :)

**Comment by l5io** [comment]
🚀 Preview for commit c0ddb7db887170e511a5876ba960bb506ac9994c at: https://64acfcbde293c44774e26fc3--layer5.netlify.app

**Comment by sayyedarib** [comment]
@Savio629 fixed the color as mentioned in the meet

**Comment by leecalcote** [comment]
Uh-oh, this PR has slipped under the radar.

There is a recent update in this area - https://github.com/layer5io/layer5/pull/4643. We can still proceed with the enhancement in this PR, if you're up to it, @sayyedarib

**Comment by sayyedarib** [comment]
Sure 
Do i have to add meshery catalog in project or something more?

**Comment by Savio629** [comment]
@sayyedarib One of the suggestion here was to add a label similar to the video label present on the [Meshery](https://meshery.io/) website

**Comment by sayyedarib** [comment]
> @sayyedarib One of the suggestion here was to add a label similar to the video label present on the [Meshery](https://meshery.io/) website

sorry for late reply ⌚ 
@Savio629 can you tell me what exactly i have to do now? Also clarify about the video label. 🤔

**Pull request closed without merging**